ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/snapshot/services/GreaterOrEqualsPageFilter.java

```diff
@@ -0,0 +1,21 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.snapshot.services;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.ldesfragment.entities.LdesFragment;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.ldesfragment.valueobjects.LdesFragmentIdentifier;
+
+import java.util.function.Predicate;
+
+public class GreaterOrEqualsPageFilter implements Predicate<LdesFragment> {
+	public static final String PAGE_NUMBER_KEY = "pageNumber";
+	private final String page;
+
+	public GreaterOrEqualsPageFilter(String lastFragment) {
+		page = LdesFragmentIdentifier.fromFragmentId(lastFragment).getFragmentPairs().get(0).fragmentValue();
```

Comment:
could you add the method getValueOfKey to the LdesFragmentIdentifier and use that one here instead of `get(0)`